### PR TITLE
Near complete frontend mvp

### PR DESCRIPTION
**Pull request by ethanimooney**
TLDR: Added most all of the front end functionality, minus JavaScript.
LINK: http://step2020-smallcity.appspot.com/

**README.md:** Made more descriptive

**index.html**: Added the main divs: main-container encompasses everything but the header, creating a 3 column grid layout. map-container will, well, contain the map. results-container is a one column, 4 row layout for the results cards and pagination buttons. filters-container is where filters will live when added.

**style.css:** Styled the page according to the mocks in go/smallcity. **Note for reviewers:** the css looks long, but there are just multiple screen breakpoints, meaning that you can simply review one breakpoint as the others are the same except for a few padding/sizing changes. 

**Note*** : Pagination buttons are of different design. I did this just to be able to get them on the page, as making them float will take more time

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -8,56 +8,570 @@ body {
...
-/*
-  Map Content Container Styling
-*/
-.map-container {
-  background-color: #e5e5e5;
-  border-radius: 30px;
-}
+    header {
+      align-items: center;
+      background-color: #b3ffb3;
```
Comment (LindseyWeiskopf) [inline src/main/webapp/style.css:19]: If you're interested (and only if you're interested), you could set these main colors to variables in the root of the css [file](https://stackoverflow.com/questions/1875852/how-can-i-define-colors-as-variables-in-css). This would make the document a little bit more readable. I'm not sure if this is good form or not but just a thought!

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -8,13 +8,38 @@
...
+          <button class="results-website-button">Visit Website</button>
+        </div>
+        <div class="results-card">
+          
+        </div>
+        <div class="results-card">
+          
+        </div>
+        <div class="pagination-container">
+          <button class="pagination-buttons">Back</button>
```
Comment (LindseyWeiskopf) [inline src/main/webapp/index.html:40]: nit: Previous and next might make a little bit more sense then forward and back

Comment (ethanimooney) [reply src/main/webapp/index.html]: I agree, changed

File: src/main/webapp/style.css (hunk 1 above)
Comment (ethanimooney) [reply src/main/webapp/style.css:19]: I think I'll leave them as-is for now but may end up changing them later once I see if I can change around the screen breakpoints. Good suggestion though, never thought of that.

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -8,13 +8,38 @@
...
+      </div>
     </header>
 
     <div class="main-container">
       <div class="map-container">
       </div>
       <div class="results-container">
+        <div class="results-card">
+          <div class="results-image">
+            image
```
Comment (shaquille-hall) [inline src/main/webapp/index.html:23]: Should this say "image"?

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -8,13 +8,38 @@
...
+        your zipcode: 
+      </div>
     </header>
 
     <div class="main-container">
       <div class="map-container">
       </div>
       <div class="results-container">
+        <div class="results-card">
+          <div class="results-image">
```
Comment (shaquille-hall) [inline src/main/webapp/index.html:22]: This won't work once you start using dynamic content. I suggest using javascript to create these page elements, and assign the correct details. Similar to what you did in your portfolio project.

File: src/main/webapp/style.css (hunk 1 above)
Comment (shaquille-hall) [reply src/main/webapp/style.css:19]: This will also prevent having to repeat the styles for the same elements. @media selectors should work like inheritance, in that one only needs to redefine the changing values, not the entire suite. I think this will make like easier going forward. 

What do you mean changing around the screen breakpoints? The values for min-device-width etc?

File: src/main/webapp/index.html (hunk 4 above)
Comment (ethanimooney) [reply src/main/webapp/index.html:22]: Yes that's the plan, I just had some in there as placeholders to get the feel for the page.

File: src/main/webapp/style.css (hunk 1 above)
Comment (ethanimooney) [reply src/main/webapp/style.css:19]: Yeah, I'm still playing around with the widths. I realize about the inheritance now, I wasn't thinking about it right, will change.

File: README.md [hunk 5]
```diff
@@ -1,9 +1,10 @@
 # SmallCity
-This directory is where we will write all of your code!
+| Created by Abakir Hanna, Ethan Mooney, and Lindsey Weiskopf
```
[the remark this answers is not in the document]
Comment (ethanimooney) [reply README.md:2]: Got it

File: src/main/webapp/index.html (hunk 3 above)
Comment (ethanimooney) [reply src/main/webapp/index.html:23]: I just had it there as placeholder text so everyone knew what it was. Will be removed once we add in the js functionality

Comment (shaquille-hall) [reply src/main/webapp/index.html:23]: Gotcha. Then please prefer a comment, that should serve the same purpose, over placeholder text that will appear in the dom.

**Comment by ethanimooney** [comment]
Fixed the issues, removed the filters section, pruned the css down **correctly** using @media this time, added support for smaller breakpoints.

**Comment by ethanimooney** [comment]
Also added the spaces in README
